lib/Conversion/CGGIToJaxite/CGGIToJaxite.cpp

```diff
@@ -150,7 +150,7 @@ struct ConvertCGGIToJaxiteLut3Op : public OpConversionPattern<cggi::Lut3Op> {
     // to jaxite to mirror jaxite API
     auto createLut3Op = rewriter.create<jaxite::Lut3Op>(
         op.getLoc(), typeConverter->convertType(op.getOutput().getType()),
-        adaptor.getC(), adaptor.getB(), adaptor.getA(), tt, serverKey, params);
+        adaptor.getA(), adaptor.getB(), adaptor.getC(), tt, serverKey, params);
```

Comment:
I was double checking here about the A, B, C ordering - I realized JaxiteOps.td doesn't document the ordering - you might want to include that in the future. (FWIW I did check that this change was correct since jaxite.lut3 expects a, b, c where a is LSB)

Reply:
You are right, the description is missed in JaxiteOps.td. Added it.